### PR TITLE
Minor edit in the upgrade documentation

### PR DESCRIPTION
**Pull request by sidkrishna**
1. The package upgrade needs to be done first to get the migration scripts so the scripts can be executed on after package upgrade. Changing the order of the steps to reflect the same. 

2. Just adding some clarity on how an existing community pack can be updated using the new pack management CLI. If someone is following the upgrade notes without prior knowledge of pack management CLI, this would help.

### CODE REVIEW

**Comment by emedvedev** [comment]
LGTM

File: docs/source/install/upgrades.rst [hunk 1]
```diff
@@ -28,10 +28,7 @@ Upgrade Procedure
 
    sudo st2ctl stop
 
-2. Run the migration script (if any). See section below for |st2|
```
Comment (dzimine) [inline docs/source/install/upgrades.rst]: Why running migration script is no longer needed/listed? I didn't try this so I don't know, but @Kami knows, so pls check with him and once he's ok I am ok.

Comment (sidkrishna) [inline docs/source/install/upgrades.rst]: @dzimine  Thanks for looking. I have changed the sequence and havn't removed step for running migration script. Basically, the migration scripts can be run once the upgrade of packages (eg. st2) has been done. The doc currently asks user to run the migration script first and then do the package upgrade which does not work.

@Kami  Could you please have a look at this PR and let me know if the changes are OK or not.

**Comment by armab** [comment]
The PR is legit, good catch :+1: 

Upgrade `st2` package first and then apply migration script.

File: docs/source/install/upgrades.rst (hunk 1 above)
Comment (sidkrishna) [inline docs/source/install/upgrades.rst]: @Kami could you please have a quick look? thanks

**Comment by Kami** [comment]
Thanks - the changes look good to me.

Will merge it into master and also cherry-pick into v2.1.

**Comment by Kami** [comment]
Ah, ignore my last statement - PR was opened against v2.1 so I will do it vice-versa (cherry pick it into master).

Anyway, thanks again and good catch :)

**Pull request merged**